packages/react/src/lib/factories.ts

```diff
@@ -91,6 +92,10 @@ type CreateShorthandFactoryConfigInner<TPropName = string> = {
   mappedArrayProp?: TPropName
 }
 export type CreateShorthandFactoryConfig = CreateShorthandFactoryConfigInner
+export type CreateShorthandFactoryResult = (
```

Comment:
Actually, the result of `createShorthandFactory` is `ShorthandFactory`

```suggestion
export type ShorthandFactory = (
```